src/metabase/driver.clj

```diff
@@ -594,3 +594,31 @@
   {:added "0.37.0" :arglists '([driver])}
   dispatch-on-initialized-driver
   :hierarchy #'hierarchy)
+
+(defmulti get-sensitive-fields
+  "Return the set of sensitive fields for the driver"
+  {:added "0.39.0" :arglists '([driver])}
+  dispatch-on-initialized-driver
+  :hierarchy #'hierarchy)
+
+(def ^:const default-sensitive-fields
```

Comment:
Even tho this _is_ a constant, I would advise against making it `^:const`. This just means it gets inlined into everywhere that uses it. So that might actually make performance worse if we end up with more code overall. It will make testing or REPL-based development with it harder since you can't `with-redefs` it, or if you change the value anything using it won't pick up the changes until you reload it

In general `^:const` makes most sense for simple constants like integers

Reply:
I guess the old code had this marked `^:const`... I would remove that

Reply:
Removed in the refactor.